modules/ting_reference/templates/ting_reference_item.tpl.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Default ting reference item template.
+ *
+ * Available variables:
+ *  - $content: Item content.
+ *  - $entity_id: The id of the entity referenced.
+ *  - $view_mode: The view mode for the reference.
+ */
+?>
+<span class="<?php print $classes; ?>" data-entity-id="<?php print $entity_id; ?>" data-view-mode="<?php print $view_mode; ?>">
```

Comment:
Perhaps it is just me being oldschool but nesting `div` inside `span` does not seem right. Is there a reason for not just using `div`s?

Reply:
Changed to "div".